Select_Flick_Week2/ShareViewController.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  ShareViewController.swift
+//  Select_Flick_Week2
+//
+//  Created by admin on 7/13/16.
+//  Copyright © 2016 koyot3. All rights reserved.
+//
+
+import UIKit
+import Social
+
+class ShareViewController: UIViewController {
+
+    @IBOutlet weak var statusField: UITextView!
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        let shareBtn = UIBarButtonItem(title: "Share", style: UIBarButtonItemStyle.Plain, target: self, action: #selector(addTapped))
+        self.navigationItem.setRightBarButtonItem(shareBtn, animated: true)
+    }
+    func addTapped(sender:AnyObject){
+        if statusField.text != "" {
+            let actionSheet = UIAlertController(title: "", message: "Share your Note", preferredStyle: UIAlertControllerStyle.ActionSheet)
+            // Configure a new action for sharing the note in Twitter.
+            let tweetAction = UIAlertAction(title: "Share on Twitter", style: UIAlertActionStyle.Default) { (action) -> Void in  
+                    // Check if sharing to Twitter is possible.
+                    if SLComposeViewController.isAvailableForServiceType(SLServiceTypeTwitter) {
+                        let twitterComposeVC = SLComposeViewController(forServiceType: SLServiceTypeTwitter)
+                        
+                        twitterComposeVC.setInitialText("\(self.statusField.text)")
+                        
+                        self.presentViewController(twitterComposeVC, animated: true, completion: nil)
+                    }
+                    else {
+                        self.showAlertMessage("You are not logged in to your Twitter account.")
+                    }
+                
+            }
+            
```

Comment:
my bad behaviour! I'm quite familiar with Visual Studio, it has "Auto format" function to beauty up codes. I rely on IDE too much, I think I should try do it manually.